package.json

```diff
@@ -132,6 +132,10 @@
   "engines": {
     "node": "18.* || 20.* || >= 21"
   },
+  "volta": {
+    "node": "18.19.0",
```

Comment:
Can we use Node 21?

Reply:
I suppose, but convention has been to keep local environment on min-supported so you don't accidentally use a feature that's not in your min-supported node.

(exception being libraries that don't need node at all, such as browser-only libraries)

Reply:
We have CI tests for all of our supported Node versions. I would prefer to rely on those to catch issues.

Reply:
you got it :tada: